gms/feature_service.cc

```diff
@@ -396,4 +396,12 @@ future<> feature_service::enable(std::set<std::string_view> list) {
     });
 }
 
+std::set<std::string_view> feature_service::supported_custom_classes() const {
```

Comment:
`feature_service` is not the right place to put this. This module is used for tracking features which need to be supported by all nodes in a Scylla cluster before they can be used by any node. This is not the right way to use this class.

If you want to have a bool which is only be set to `true` after all nodes support the feature (which is the right approach for making support conditional on this), then you should just add a new `gms::feature` field in the definition of the `gms::feature_service` class and it will work automatically.